scss/main.scss

```diff
@@ -102,3 +102,5 @@ nav {
     }
   }
 }
+
+@import 'responsive';
```

Comment:
@jehna that's why it's better to put these responsive directives _inside_ other CSS elements, like I explained above